setup.cfg

```diff
@@ -20,10 +20,17 @@ universal = 1
 add_ignore = D401
 
 [pycodestyle]
-exclude = docs/conf.py
+exclude = docs/conf.py, invenio_app_ils/config.py
```

Comment:
easier than adding # noqa  in every string module path

Reply:
true, but the config.py is a massive file... can we simply exclude the line-length in that file?